resources/lang/en/events/injury/desert.json

```
@@ -0,0 +1,51 @@
+[
+   {
+        "event_id": "dst_injury_bruisessore_anymultiage1",
+        "location": ["desert"],
+        "season": ["any"],
+        "weight": 20,
+        "event_text": "m_c was out in the territory and took a bad tumble over a rock hidden by the dusty ground.",
+        "m_c": {
+            "age": ["adolescent", "young adult", "adult", "senior adult", "senior"]
+        },
+        "injury": [
+            {
+                "cats": ["m_c"],
+                "injuries": ["bruises", "sore"]
+            }
+        ]
+    },
+   {
+        "event_id": "dst_injury_scrapessmallcuts_appmultitrait1",
+        "location": ["desert"],
+        "season": ["any"],
+        "weight": 20,
+        "event_text": "m_c dashes off from {PRONOUN/m_c/poss} mentor and tumbles face first into a cactus, thankfully the medicine cat is able to remove the spines with little difficulty.",
```

```suggestion
        "event_text": "m_c dashed off from {PRONOUN/m_c/poss} mentor and tumbled face first into a cactus. Thankfully, the medicine cat was able to remove the spines with little difficulty.",
```

This event relies on the apprentice having a mentor, and also that the clan has a medicine cat. I don't know if we can constrain for those things here.